packages/search-service/src/helpers.ts

```diff
@@ -0,0 +1,94 @@
+import * as _ from 'lodash';
+const fetch = require( 'node-fetch' );
+import https from 'https';
+
+import { HttpsProxyAgent } from 'https-proxy-agent';
+global.Headers = fetch.Headers;
+
+class IndexHelper {
+    private static UserGroupHelperInstance: IndexHelper;
+    constructor () { }
+    public static indexHelper () {
+        if ( !IndexHelper.UserGroupHelperInstance ) {
+            IndexHelper.UserGroupHelperInstance = new IndexHelper();
+        }
+        return IndexHelper.UserGroupHelperInstance;
+    }
+
+    public index ( body: any ) {
+        return new Promise( async ( resolve, reject ) => {
+            let headers = new Headers();
+            console.log( process.env.AUTH_TOKEN );
+            headers.append( `Authorization`, `${ process.env.AUTH_TOKEN }` );
```

Comment:
Remove `console.log`.

```suggestion
            headers.append( `Authorization`, `${ process.env.AUTH_TOKEN }` );
```